ios/Contacts/RNPermissionHandlerContacts.m

```diff
@@ -14,6 +16,13 @@ + (NSString * _Nonnull)handlerUniqueId {
 
 - (void)checkWithResolver:(void (^ _Nonnull)(RNPermissionStatus))resolve
                  rejecter:(void (__unused ^ _Nonnull)(NSError * _Nonnull))reject {
+#if TARGET_OS_TV
```

Comment:
You mix up `NSError` based and string based rejection. Maybe you can keep it to the second?

```objc
#if TARGET_OS_TV
    NSError * error = [[NSError alloc]
                       initWithDomain:@"RNPermissions"
                       code:1
                       userInfo:@{NSLocalizedDescriptionKey: @"Contacts not available on tvOS"}];
    return reject(error);
#elif

// ⬇️

#if TARGET_OS_TV
  return reject(@"cannot_check_permission", @"Contacts not available on tvOS", nil);
#elif
```